spec/features/volunteer_adds_a_case_contact_spec.rb

```diff
@@ -1,31 +1,46 @@
-require "rails_helper"
+require 'rails_helper'
 
-RSpec.feature "volunteer adds a case contact", type: :feature do
-  scenario "is successful" do
+RSpec.describe 'volunteer adds a case contact', type: :feature do
+  it 'is successful' do
     volunteer = create(:user, :volunteer, :with_casa_cases)
     volunteer_casa_case_1 = volunteer.casa_cases.first
 
     sign_in volunteer
 
     visit new_case_contact_path
-    within "select#casa_case_id" do
-      select volunteer_casa_case_1.case_number
-    end
-    within "select#contact_type" do
-      select "School"
-    end
 
-    within "select#duration_minutes" do
-      select "60 minutes"
-    end
+    select volunteer_casa_case_1.case_number, from: 'casa_case_id'
+    select 'School', from: 'contact_type'
+    select '60 minutes', from: 'duration_minutes'
+    fill_in 'case_contact_occurred_at', with: '04/04/2020'
 
-    # find(:css, "#case_contact_occurred_at".)
-    fill_in "case_contact_occurred_at", with: "04/04/2020"
-
-    click_on "Submit"
+    click_on 'Submit'
 
     expect(CaseContact.first.casa_case_id).to eq volunteer_casa_case_1.id
-    expect(CaseContact.first.contact_type).to eq "school"
+    expect(CaseContact.first.contact_type).to eq 'school'
     expect(CaseContact.first.duration_minutes).to eq 60
   end
+
+  it 'chooses other' do
+    volunteer = create(:user, :volunteer, :with_casa_cases)
+    volunteer_casa_case_1 = volunteer.casa_cases.first
+
+    sign_in volunteer
+
+    visit new_case_contact_path
+
+    select volunteer_casa_case_1.case_number, from: 'casa_case_id'
+    select 'School', from: 'contact_type'
+    expect(page).not_to have_selector '#case_contact_other_type_text'
+    select '90 minutes', from: 'duration_minutes'
+    fill_in 'case_contact_occurred_at', with: '04/04/2020'
+    select 'Other', from: 'contact_type'
+
+    click_on 'Submit'
+
+    expect(CaseContact.first.casa_case_id).to eq volunteer_casa_case_1.id
+    expect(CaseContact.first.contact_type).to eq 'other'
+    expect(CaseContact.first.other_type_text).to eq nil
```

Comment:
Is it allowable for the `other_type_text` to be `nil` when `other` is selected? Perhaps this test is a case in point of why a model validation might be a good idea. But I'm surprised the test runner is letting this pass, tbh! 😃